recipes/py7zr/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "py7zr" %}
+{% set version = "0.16.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0ab402f73fc8cc41f2a5523436ae53dec6ff612597d9168ed80cfa1574d27fe0
+
+build:
+  number: 0
```

If this is pure Python can't this be `noarch: python`?
```suggestion
  number: 0
  noarch: python
```